// tailwindcss/src/featureFlags.js
ter: true }
        Property { name: "handle"; type: "QQmlComponent"; isPointer: true }
        Property { name: "incrementControl"; type: "QQmlComponent"; isPointer: true }
        Property { name: "decrementControl"; type: "QQmlComponent"; isPointer: true }
        Property { name: "__scrollbar"; type: "QQmlComponent"; isPointer: true }
        Property { name: "__externalScrollBars"; type: "bool" }
        Property { name: "__scrollBarSpacing"; type: "int" }
        Property { name: "__scrollBarFadeDelay"; type: "int" }
        Property { name: "__scrollBarFadeDuration"; type: "int" }
        Property { name: "__stickyScrollbars"; type: "bool" }
    }
    Component {
        prototype: "QQuickFocusScope"
        name: "QtQuick.Controls/TextField 1.0"
        exports: ["QtQuick.Controls/TextField 1.0"]
        exportMetaObjectRevisions: [0]
        isComposite: true
        defaultProperty: "data"
        Property { name: "inputMethodComposing"; type: "bool"; isReadonly: true }
        Property { name: "selectByMouse"; type: "bool" }
        Property { name: "menu"; type: "QQmlComponent"; isPointer: true }
        Property { name: "acceptableInput"; type: "bool"; isReadonly: true }
        Property { name: "activeFocusOnPress"; type: "bool" }
        Property { name: "canPaste"; type: "bool"; isReadonly: true }
        Property { name: "canRedo"; type: "bool"; isReadonly: true }
        Property { name: "canUndo"; type: "bool"; isReadonly: true }
        Property { name: "textColor"; type: "QColor" }
        Property { name: "cursorPosition"; type: "int" }
        Property { name: "cursorRectangle"; type: "QRectF"; isReadonly: true }
